test/test_primal_dual_objective.py

```diff
@@ -71,11 +76,14 @@ def test_dual_obj():
     disp = init_gap + surface.heights() + offset
     init_pressure = substrate.evaluate_force(disp)
 
-    res = optim.minimize(system.dual_objective(offset, gradient=True),
-                         init_pressure,
-                         method='L-BFGS-B', jac=True,
-                         bounds=bnds,
-                         options=dict(gtol=1e-8, ftol=1e-20))
+    # res = optim.minimize(system.dual_objective(offset, gradient=True),
+    #                      init_pressure,
+    #                      method='L-BFGS-B', jac=True,
+    #                      bounds=bnds,
+    #                      options=dict(gtol=1e-8 * system.area_per_pt,
+    #                                   ftol=1e-20))
+    res = system.dual_minimize_proxy(x0=disp, offset=offset, gtol=1e-8)
```

Comment:
Is that lbfgsb under the hood ?

Reply:
bugnicourt for both primal and dual minimize proxy

Reply:
I was asking because you replaced lbfgsb with that. 
note that a consequence of this is that now you test the gradient and the hessian whereas before you tested gradient and objective. I wonder wether we should simply parametetrize the minimizer.